2.0 release content (#355)
apache/airflow-site: landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md

For more information, read the [Scheduler HA documentation](http://airflow.apache.org/docs/apache-airflow/stable/scheduler.html#running-more-than-one-scheduler).

## Task Groups (AIP-34)

SubDAGs were commonly used for grouping tasks in the UI, but they had many drawbacks in their execution behaviour (primarirly that they only executed a single task in parallel!) To improve this experience, we’ve introduced "Task Groups": a method for organizing tasks which provides the same grouping behaviour as a subdag without any of the execution-time drawbacks.

SubDAGs will still work for now, but we think that any previous use of SubDAGs can now be replaced with task groups. If you find an example where this isn't the case, please let us know by opening an issue on GitHub

For more information, check out the [Task Group documentation](http://airflow.apache.org/docs/apache-airflow/stable/concepts.html#taskgroup).

## Refreshed UI

We've given the Airflow UI [a visual refresh](https://github.com/apache/airflow/pull/11195) and updated some of the styling.

![Airflow 2.0's new UI](airflow-2.0-ui.gif)

> Check out [the screenshots in the docs](http://airflow.apache.org/docs/apache-airflow/stable/ui.html) for more.

Reviewer: ```suggestion
We have also added an option to auto-refresh task states in Graph View so you no longer need to continuously press the refresh button :).

Check out [the screenshots in the docs](http://airflow.apache.org/docs/apache-airflow/stable/ui.html) for more.
```

## Smart Sensors for reduced load from sensors (AIP-17)

> If you make heavy use of sensors in your Airflow cluster, you might find  that sensor execution takes up a significant proportion of your cluster even with "reshedule" mode. To improve this,  we've added a new mode called "Smart Sensors".

Reviewer: ```suggestion
If you make heavy use of sensors in your Airflow cluster, you might find that sensor execution takes up a significant proportion of your cluster even with "reschedule" mode. To improve this,  we've added a new mode called "Smart Sensors".
```

Author: Can't apply this suggestion for some reason.

Our very own Jarek Potiuk has written about [providers in much more detail](https://www.polidea.com/blog/airflow-2-providers/) on the Polidea blog.

Docs on the [providers concept and writing custom providers](http://airflow.apache.org/docs/apache-airflow-providers/)  
Docs on the [all providers packages available](http://airflow.apache.org/docs/apache-airflow-providers/packages-ref.html)

## Security

As part of Airflow 2.0 effort, there has been a conscious focus on Security and reducing areas of exposure. This is represented across different functional areas in different forms. For example, in the new REST API, all operations now require authorization. Similarly, in the configuration settings, the Fernet key is now required to be specified.

## Configuration

Configuration in the form of the airflow.cfg file has been rationalized further in distinct sections, specifically around "core". Additionally, a significant amount of configuration options have been deprecated or moved to individual component-specific configuration files, such as the pod-template-file for Kubernetes execution-related configuration.

## Thanks to all of you

> We've tried to make as few breaking changes as possible and to provide deprecation path in the code, especially in the case of anything called in the DAG. That said,  please read throughUPDATING.md to check what might affect you. For example: r We re-organized the layout of operators (they now all live under airflow.providers.*) but the old names should continue to work - you'll just notice a lot of DeprecationWarnings that need to be fixed up.

Reviewer: ```suggestion
We've tried to make as few breaking changes as possible and to provide deprecation path in the code, especially in the case of anything called in the DAG. That said,  please read throughUPDATING.md to check what might affect you. For example: We re-organized the layout of operators (they now all live under airflow.providers.*) but the old names should continue to work - you'll just notice a lot of DeprecationWarnings that need to be fixed up.
```

> Thank you so much to all the contributors who gotus to this point, in no particular order: Kaxil Naik, Daniel Imberman, Jarek Potiuk, Tomek Urbaszek, Kamil Breguła, Gerard Casas Saez, Xiaodong DENG, Kevin Yang, James Timmins, Yingbo Wang, Qian Yu, Ryan Hamilton and the 100s of others who keep making Airflow better for everyone.

Reviewer: ```suggestion
Thank you so much to all the contributors who got us to this point, in no particular order: Kaxil Naik, Daniel Imberman, Jarek Potiuk, Tomek Urbaszek, Kamil Breguła, Gerard Casas Saez, Xiaodong DENG, Kevin Yang, James Timmins, Yingbo Wang, Qian Yu, Ryan Hamilton and the 100s of others who keep making Airflow better for everyone.
```
